File: back/service/src/main/java/org/webtree/mydata/service/PersonalDataService.java
package org.webtree.mydata.service;

import org.springframework.stereotype.Service;
import org.webtree.mydata.domain.User;
import reactor.core.publisher.Flux;

import java.util.Set;

@Service
public class PersonalDataService {
    public Flux findBy(User user, Set tags) {
        throw new RuntimeException();
    }
}

File: api/src/main/kotlin/org/webtree/mydata/api/service/PersonalDataService.kt
package org.webtree.mydata.api.service

import org.webtree.mydata.api.domain.PersonalData
import org.webtree.mydata.api.domain.Tag
import org.webtree.mydata.api.domain.User
import reactor.core.publisher.Flux

class PersonalDataService {
    fun findBy(user: User, tags: Set<Tag>): Flux<PersonalData> {
        TODO()
    }
}